client/src/pages/PackageDetailsScreen/MatchCard/MatchDetails/MatchDetails.tsx

```diff
@@ -0,0 +1,74 @@
+import { Typography, Button, Tag } from 'antd';
+import { TrophyOutlined, EnvironmentOutlined } from '@ant-design/icons';
+import styles from './match-details.module.scss';
+import { formattedDate } from '@/utils/date.utils';
+
+const { Text } = Typography;
+
+interface MatchDetailsProps {
+    homeTeamImage: string;
+    awayTeamImage: string;
+    homeTeam: string;
+    awayTeam: string;
+    stadium: string;
+    league: string;
+    matchDate: string;
+    price: number;
+    linkForTicket: string;
```

Comment:
```suggestion
match: Match
```
no need for this many props if they are all related to match and it is a `MatchDetails` component